tuxedo-core/src/executive.rs

```diff
@@ -898,7 +898,7 @@ mod tests {
                         "858174d563f845dbb4959ea64816bd8409e48cc7e65db8aa455bc98d61d24071",
                     ),
                     extrinsics_root: array_bytes::hex_n_into_unchecked(
-                        "7ceffb73687cb9af3ad2f9a0c544a216df70894b03da3ceb57ead37bd6b51be0",
+                        "b3dd85732bb51b79d7c5c3518cfb1b202210743b9dc7b8ccf848f9a66e0a0f6b",
```

Comment:
What is this change about ? Just curious.

Reply:
The extrinsics root is the merkle tree root that you get when you build all of the extrinsics into a merkle tree. Since the extrinsics encoding changed here (to have the length prepended) the root also changed.

Notice it only changed in this test where there is actually an extrinsic in the block. The other tests where the block is empty still have the same root as before. This is good and confirms this test only changed because of the new encoding, not some other weird bug I introduced.

I didn't manually calculate that new root or expect that exact hash. I just noticed that it changed (because the test failed) decided it made sense that it changed (as explained above) and copied the new root. AFAICT that's what the FRAME tests do too. That's what I did to get the roots when I first wrote those tests anyway.

This does feel a tiny bit questionable, but I rationalized it saying the test still adds value because it lets us know any time the value changes. And if it changes when we don't expect it to, that means we have a problem. If it changes when we do expect it to (like now) we can just update the value.